en/restart-a-tidb-cluster.md

```diff
@@ -6,67 +6,37 @@ category: how-to
 
 # Restart a TiDB Cluster in Kubernetes
 
-This document describes how to forcibly restart a TiDB cluster in the Kubernetes cluster, including restarting a Pod, restarting all Pods of a component, and restarting all Pods of the TiDB cluster.
+If you find that the memory leak occurs in a Pod of the TiDB cluster during use, you need to restart the cluster. This document describes how to logoff a Pod and restart a TiDB cluster gracefully.
 
-> **Note:**
+> **Warning:**
 >
-> TiDB Operator v1.0.x only supports restarting the Pod in a forcible way.
->
-> - If the PD Pod being restarted is the PD leader, its leadership does not automatically transfer, which can cause the PD service to be temporarily unavailable.
-> - If the TiKV Pod being restarted contains a Region leader of the TiKV cluster, its leadership does not automatically transfer, which can cause the requests of accessing the corresponding data to fail.
-> - Restarting the TiDB Pod can cause the requests of accessing this Pod to fail.
-
-## Forcibly restart a Pod
-
-Execute the following command to forcibly restart a Pod:
-
-{{< copyable "shell-regular" >}}
-
-```shell
-kubectl delete pod -n <namespace> <pod-name>
-```
-
-## Forcibly restart all Pods of a component
-
-Execute the following command to check the Pod list of a component:
-
-{{< copyable "shell-regular" >}}
+> It is not recommended to manually remove a Pod in the TiDB cluster without graceful restart in a production environment, because this might lead to some request failures of accessing the TiDB cluster though the `StatefulSet` controller pulls the Pod up again.
 
-```shell
-kubectl get pod -n <namespace> -l app.kubernetes.io/component=<component-name>
-```
+## Enable the configurations
 
-Execute the following command to forcibly restart all Pods of a component:
+To activate the graceful logoff feature, you need to enable some related configurations in TiDB Operator. These configurations are disabled by default. Take the following steps to manually turn them on.
 
-{{< copyable "shell-regular" >}}
-
-```shell
-kubectl delete pod -n <namespace> -l app.kubernetes.io/component=<component-name>
-```
+1. Edit the `values.yaml` file.
 
-> **Note:**
->
-> - To forcibly restart all Pods of the `PD`, `TiDB`, or `TiKV` component, replace `<component-name>` in the above commands with `pd`, `tidb`, or `tikv` respectively.
-> - Replace `<namespace>` in the above commands with your own namespace.
+    Enable the `Operator Webhook` feature:
 
-## Forcibly restart all Pods of the TiDB cluster
+    ```yaml
+    admissionWebhook:
+      create: true
+      mutation:
+        pods: true
```

Comment:
These 2 lines should be removed.

Reply:
@Yisaer Shall we remove them from the Chinese doc as well?

Reply:
Yes, it has already been removed.